### DISCUSSION TITLE
Google translate 500 (Internal Server Error) Any advice ?

### DISCUSSION THREAD

**Discussion by yamlv** [chatwoot/chatwoot#9063] [Self hosted]
Hi,
Added Google translate integration, on right click translate text i see it succesfully receives API request in google cloud, but in chrome console i see error and it does not translates anything.

Chatwoot version v3.6.0, ubuntu (it also was not working on 3.52)

In chrome console i got this as soon i click translate in chatwoot....

message.js:104 
POST https://my..... domain ...... com/api/v1/accounts/3/conversations/834/messages/11949/translate 500 (Internal Server Error)
(anonymous) @ xhr.js:244
xhr @ xhr.js:47
$ @ dispatchRequest.js:46
value @ Axios.js:143
(anonymous) @ Axios.js:184
(anonymous) @ bind.js:5
value @ message.js:104
(anonymous) @ messageTranslateActions.js:6
c @ videojs.record.js:2868
(anonymous) @ videojs.record.js:3088
S.forEach.e.<computed> @ videojs.record.js:2921
Xo @ application-bf58184cd8b47c9f46e9.js:2
o @ application-bf58184cd8b47c9f46e9.js:2
(anonymous) @ application-bf58184cd8b47c9f46e9.js:2
(anonymous) @ application-bf58184cd8b47c9f46e9.js:2
translateMessage @ messageTranslateActions.js:4
(anonymous) @ vuex.js:778
p.dispatch @ vuex.js:497
dispatch @ vuex.js:416
handleTranslate @ MessageContextMenu.vue:211
dn @ vue.common.prod.js:1404
n @ vue.common.prod.js:657
dn @ vue.common.prod.js:1404
e.$emit @ vue.common.prod.js:2723
click @ menuItem.vue:1
dn @ vue.common.prod.js:1404
n @ vue.common.prod.js:657
i._wrapper @ vue.common.prod.js:3521
Show 19 more frames
Show less

Any advice ? 

Thank you, for great chat!

**Comment by stephanedupont** [comment]
Same here... Did you manage to solve this?

**Comment by VLadislav9607** [comment]
same here any solutions?

<img width="349" height="196" alt="Знімок екрана 2025-11-07 о 09 49 50" src="https://github.com/user-attachments/assets/05badbbb-b342-477f-8722-3e7501a6ba0c" />
